airflow/models/dagcode.py

```diff
@@ -169,27 +171,30 @@ def get_code_by_fileloc(cls, fileloc):
         :param fileloc: file path of a DAG
         :return: source code as string
         """
-        return DagCode(fileloc).code()
+        return cls.code(fileloc)
 
-    def code(self):
+    @classmethod
+    def code(cls, fileloc):
```

Comment:
I don't like this method name. It describes what he does. It does not contain any verb. However, this is not related to this PR.

Reply:
I have a separate PR to improve the code in this file, that I have targetted against Master but yeah that is not the priority for now.